tests/unit/phpDocumentor/Descriptor/ClassDescriptorTest.php

```diff
@@ -377,6 +377,7 @@ public function testGetMagicMethods()
         $methodMock->shouldReceive('getDescription')->andReturn($description);
         $methodMock->shouldReceive('getResponse')->andReturn($response);
         $methodMock->shouldReceive('getArguments')->andReturn($arguments);
+        $methodMock->shouldReceive('isStatic')->andReturn(true);
```

Comment:
Please do add this to a new test. It is unclear to me now if the original behavior was not changed unintended.
It looks like is't ok, but I want to be sure.